public/test/elements-with-checkout-sessions/code.html

```diff
@@ -0,0 +1,11 @@
+<script src="https://js.stripe.com/basil/stripe.js"></script>
+
+<p id="checkout-express-checkout-element"></p>
+<label>Email: <input type="text" id="email" /></label>
```

Comment:
```suggestion
<label>Email: <input type="email" id="email"></label>
```

Two minor suggestions:

1. Remove the trailing slash (convention in this codebase is to not self-close HTML elements with no closing tag)
2. Change the `<input>`'s `type` to `email` (unless there's a good reason not to?)